railties/lib/rails/application/configuration.rb

```diff
@@ -357,7 +357,9 @@ def load_defaults(target_version)
           self.yjit = !Rails.env.local?
 
           if respond_to?(:action_controller)
-            action_controller.escape_json_responses = false
+            ActionController.deprecator.silence do
+              action_controller.escape_json_responses = false
+            end
```

Comment:
I don't think this is correct, because here `action_controller.escape_json_responses =` isn't the method you defined, it's an `OrderedOption`, the value will be forwarded to `ActionController:Base` from an initializer.

Reply:
Also `false` doesn't emit a warning anyway.

Reply:
This one wasn't addressed.